### DISCUSSION TITLE
Não recebimento dos benefícios do GitHub Educação / Not receiving GitHub Education benefits

### DISCUSSION THREAD

**Discussion by silvio-cast** [community/community#138224] [GitHub Education]
### Hi everyone,

Boa tarde,
Candidatei-me ao GitHub Education no dia 3 de setembro e, apesar de ter sido aprovado nessa data, ainda não recebi os benefícios.

Good afternoon,
I applied for GitHub Education on September 3rd and, despite being approved on that date, I have not yet received the benefits.

**Comment by davevad93** [comment]
Hi @silvio-cast ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @silvio-cast was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @silvio-cast was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack) and select one of the available offers, if you can access them you are fine, if not I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887
